cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -123,33 +145,33 @@ if(is_mpi)
     if(NOT MPI_CXX_FOUND)
       # Fix concurrent output to console
       execute_process(
-          COMMAND
```

Comment:
Actually this indentation is by design. The rule is 2 for blocks (e.g. `if` or `foreach`):
```cmake
if(A)
  # 2 spaces
  if(B)
    # + 2 spaces
    foreach(x ${y})
      # + 2 spaces
      message("x = ${x}")
    endforeach()
  endif()
endif()
```

and 4 spaces for arguments:
```cmake
configure_file("${x}" "${y}" @ONLY) # short enough

# longer
configure_file(
    # 4 spaces
    "/path/to/x" "/path/to/y" @ONLY
)

# very long
configure_file(
    # 4 spaces
    "/very/long/path/to/x"
    "/very/long/path/to/y"
    @ONLY
)
```

Mix:

```cmake
if(A)
  # +2 for block
  execute_process(
      # +4 for arguments
      COMMAND "${CMAKE_COMMAND}" -E echo "Hello"
      WORKING_DIRECTORY "/path/to/working/directory"
  )
  message("DONE")
  if(B)
    # +2 for block
    message("With B")
  endif()
endif()
```